bellbot_gui/srv/FeedbackDone.srv

```diff
@@ -0,0 +1 @@
+---
```

Comment:
Actually, there is already an Empty Service defined in std_srvs: http://wiki.ros.org/std_srvs so no need for this here.